openmc/mgxs/__init__.py

```diff
@@ -593,7 +594,208 @@
     2.4000e8, 2.8000e8, 3.2000e8, 3.6000e8, 4.0000e8,
     4.4000e8, 4.8000e8, 5.2000e8, 5.6000e8, 6.0000e8,
     6.4000e8, 6.8000e8, 7.2000e8, 7.6000e8, 8.0000e8,
-    8.4000e8, 8.8000e8, 9.2000e8, 9.6000e8, 1.0000e9,])
+    8.4000e8, 8.8000e8, 9.2000e8, 9.6000e8, 1.0000e9])
+GROUP_STRUCTURES['SCALE-999'] = np.array([
+    1.000e-5, 1.000e-4, 5.000e-4, 7.500e-4, 1.000e-3,
```

Comment:
I am looking at page 39 of the reference provided https://www.nrc.gov/docs/ML1218/ML12184A002.pdf and it looks to me like the lowest energy bin is 1.000e-4 instead of 1.000e-5